development/compiling/introduction_to_the_buildsystem.rst

```diff
@@ -124,63 +124,56 @@ you can build them yourself).
 Aside from that, there are a few standard options that can be set in all
 build targets, and which will be explained below.
 
-.. _doc_introduction_to_the_buildsystem_tools:
+.. _doc_introduction_to_the_buildsystem_target:
 
-Tools
------
+Target
+------
 
-Tools are enabled by default in all PC targets (Linux, Windows, macOS),
-disabled for everything else. Disabling tools produces a binary that can
-run projects but that does not include the editor or the project
-manager.
+Target controls if the editor is contained and used debug flags.
+All builds are optimized. Each mode means:
 
-::
+-  **editor**: Build with editor, optimized, with debugging code
+-  **template_debug**: Build with C++ debugging symbols
+-  **template_release**: Build without symbols
```

Comment:
```suggestion
-  **template_debug**: Build with C++ debugging symbols (defines: `DEBUG_ENABLED`, `-O2`/`/O2`)
-  **template_release**: Build without symbols (defines: `-O3`/`/O2`)
```

Reply:
@clayjohn Backticks should be doubled in rSt syntax: \`\`

Reply:
```suggestion
-  **template_debug**: Build with C++ debugging symbols (defines: ``DEBUG_ENABLED``, ``-O2``/``/O2``)
-  **template_release**: Build without symbols (defines: ``-O3``/``/O2``)
```